tests/unit/seal/test_prompt_templates.py

```
@@ -147,30 +151,32 @@ def mock_time(self, monkeypatch):
         def mock_time_func():
             return current_time
 
-        monkeypatch.setattr('time.time', mock_time_func)
+        monkeypatch.setattr("time.time", mock_time_func)
         return current_time
 
     @pytest.mark.asyncio
     async def test_template_caching(self):
         """Test that templates are properly cached."""
         import time
```

Flake8 (7.2.0)

[error] 'time' imported but unused

(F401)